pkg/validator/properties.go

```diff
@@ -0,0 +1,18 @@
+package validator
+
+import (
+	"github.com/magiconair/properties"
+)
+
+type PropValidator struct{}
+
+// Validate implements the Validator interface by attempting to
+// parse a byte array of properties
+func (pv PropValidator) Validate(b []byte) (bool, error) {
+	l := &properties.Loader{Encoding: properties.UTF8}
+	_, err := l.LoadBytes(b)
+	if err != nil {
+		return false, err
```

Comment:
Is it possible to show the line and column where the error was found in the file? Local testing shows that is not being included in the error output:

```
    × /tests/fixtures/subdir2/bad.properties
        error: circular reference in:
key=${key}
```

Also in the above error message `key=${key}` is on the next line and not indented properly

Reply:
I will look into this.

Reply:
I have fixed the indentation issue, now error message is as follows:
```
    × test/fixtures/subdir2/bad.properties
        error: circular reference in:
               key=${key}
```
I checked the issue with line and column not being printed out.

Actually errors like `=key` throws error with the line context
```
    × test/fixtures/subdir2/bad.properties
        error: properties: Line 48: "key"
```
This error is caught as part of parsing process in `magiconair/properties` and has the Line context.
https://github.com/magiconair/properties/blob/c9a06e8f8f0164e4e16c0d5c4793cbed4ac90264/parser.go#L58

But for `key=${key}`, this error is caught during expansion process and the line context is lost at that point.
https://github.com/magiconair/properties/blob/c9a06e8f8f0164e4e16c0d5c4793cbed4ac90264/properties.go#L766
To have this feature we might need changes in `magiconair/properties` 

How do you suggest we proceed here @kehoecj

Reply:
Thank you for the detailed analysis! Let's proceed with your current implementation and you (or I) can create an issue in the `magiconair/properties` project to include line context in expansion errors

Reply:
I have a couple tests I want to run locally with the changes to the standard report padding - if all pass I'll resolve this and get this PR merged

Reply:
Sure, Please let me know of any findings.